scst-store/using-encryption-and-connection.md

```diff
@@ -119,8 +87,19 @@ Run this command in a separate terminal window.
 
 ### <a id='mod-etchost'></a>Modifying your `/etc/hosts` file
 
-To add the following entry to `/etc/hosts`, run:
+The following script illustrates how to add a new local entry to `/etc/hosts`:
 
-```console
-127.0.0.1 metadata-store-app.metadata-store.svc.cluster.local
+```bash
+METADATA_STORE_DOMAIN="metadata-store-app.metadata-store.svc.cluster.local"
+
+# delete any previously added entry
+sudo sed -i '' "/$METADATA_STORE_DOMAIN/d" /etc/hosts
+
+echo "127.0.0.1 $METADATA_STORE_DOMAIN" | sudo tee -a /etc/hosts > /dev/null
+```
+
+Finally, set the target 
+
+```bash
+tanzu insight config set-target https://$METADATA_STORE_DOMAIN --ca-cert insight-ca.crt
```

Comment:
we should specify a port with the url. In the port forwarding section you are forwarding 8443